LadybugTools_oM/ExecuteCommands/RunUTCIHeatPlotCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Drawing;
+using BH.oM.Adapter;
+using System.ComponentModel;
+
+namespace BH.oM.LadybugTools
+{
+    [Description("Command that, when executed with the LadybugToolsAdapter, simulates UTCI values and outputs a heatmap.\nOutput is a string of either the path to the image (if OutputLocation is not set) or the base 64 string representation of it.")]
+    public class RunUTCIHeatPlotCommand : IExecuteCommand
+    {
+        [Description("The path to an EPW file.")]
+        public virtual string EpwFile { get; set; } = "";
+
+        [Description("An opaque energy material to use as the ground material in the simulation.")]
+        public virtual IEnergyMaterialOpaque GroundMaterial { get; set; } = null;
+
+        [Description("An opaque energy material to use as the shade material in the simulation.")]
+        public virtual IEnergyMaterialOpaque ShadeMaterial { get; set; } = null;
+
+        [Description("A typology to use in the simulation containing any shelters and evaporative cooling effects. TargetWindSpeed is not used in this simulation")]
+        public virtual Typology Typology { get; set; } = null;
+
+        [Description("A wind speed multiplier to modify the wind speed by. Default is 1.")]
+        public virtual double WindSpeedMultiplier { get; set; } = 1;
+
+        [Description("A list of 10 colours to use for each UTCI category, leave empty to use the default UTCI colours.")]
```

Comment:
Colours in what format? Hexadecimal? English words? E.G. should I put `#FF0000` or `Red` (or `red`)?

Consider making its own object with ten colour inputs and maybe use the system colour choices if the options are broad enough. Reach out if this doesn't make sense, happy to explain in more detail.

Reply:
Let's take out the colour functionality for now. This is going to need a lot more thought.

Reply:
Colour functionality fixed, added new method that converts between hexadecimal and `System.Drawing.Color`.